basics/node.md

```diff
@@ -0,0 +1,26 @@
+# Interoperating with Node and NPM
+
+While Deno is pretty powerful itself, many people will want to leverage code and
+libraries that are built for [Node](https://nodejs.org/), in particular the
+large set of packages available on the [NPM](https://npmjs.com/) registry.
+
+There are currently two ways to do this:
+
+- Using [Node specifiers](./node/node_specifiers.md)
```

Comment:
```suggestion
- Using [npm specifiers](./node/node_specifiers.md)
```

@bartlomieju @dsherret we should really clarify what we are calling them... "Node specifiers" feels wrong, especially because they start with `npm`.

Reply:
Also other places they are referred to as npm specifiers (like in the section below)